src/ucp/wireup/select.c

```diff
@@ -1170,32 +1196,19 @@ ucp_wireup_add_tag_lane(ucp_wireup_select_ctx_t *select_ctx,
 
     /* Do not add tag offload lane, if selected tag lane score is lower
      * than AM score. In this case AM will be used for tag macthing. */
-    status = ucp_wireup_select_transport(ep, select_ctx->address_list,
-                                         select_ctx->address_count,
-                                         &criteria, -1, -1, -1, -1, 0,
-                                         &select_info);
+    status = ucp_wireup_select_transport(select_ctx, &criteria, -1, -1,
+                                         -1, -1, 0, &select_info);
     /* Break if: */
```

Comment:
no need in these comments now (others below as well). The original comment before   `ucp_wireup_select_transport` is enough

Reply:
done